modules/installation-configuration-parameters.adoc

```diff
@@ -754,6 +754,10 @@ belong to the same region as the cluster. This is required for regions that requ
 |An existing Route 53 private hosted zone for the cluster. You can only use a pre-existing hosted zone when also supplying your own VPC. The hosted zone must already be associated with the user-provided VPC before installation. Also, the domain of the hosted zone must be the cluster domain or a parent of the cluster domain. If undefined, the installation program creates a new hosted zone.
 |String, for example `Z3URY6TWQ91KVV`.
 
+|`platform.aws.hostedZoneRole`
+|An Amazon Resource Name (ARN) for an existing IAM Role in the account containing the specified hosted zone. The installation program and cluster operators will assume this role when performing operations on the hosted zone.  
```

Comment:
1. The `platform.aws.hostedZoneRole` is designed for Shared VPC scenario, and in the [account configuration section](https://github.com/openshift/openshift-docs/pull/64114/files#diff-b52183dc0e7629a553c56e12c5a17a12eb83300427c09229e0dbfe8b7fcd09c2R6), we used word `shared VPC`, I would suggest to mention shared VPC in this part, to correspond to the former.

2. As my comment, I think we can move trust policy and permission content to here.

Reply:
Good point, I've made that addition.